crates/solidity/testing/perf/src/tests/init_bindings.rs

```diff
@@ -16,18 +17,18 @@ pub fn setup() -> ParseOutput {
     built_ins
 }
 
-pub fn run(built_ins: ParseOutput) -> Bindings {
-    let mut bindings = create_with_resolver(SOLC_VERSION, Arc::new(NoOpResolver {}));
+pub fn run(built_ins: ParseOutput) -> BindingGraph {
+    let mut binding_graph = create_with_resolver(SOLC_VERSION, Rc::new(NoOpResolver {}));
 
-    bindings.add_system_file("built_ins.sol", built_ins.create_tree_cursor());
+    binding_graph.add_system_file("built_ins.sol", built_ins.create_tree_cursor());
 
-    bindings
+    binding_graph
 }
 
 struct NoOpResolver;
 
-impl PathResolver for NoOpResolver {
-    fn resolve_path(&self, _context_path: &str, path_to_resolve: &str) -> Option<String> {
-        Some(path_to_resolve.to_string())
+impl PathResolver<KindTypes> for NoOpResolver {
+    fn resolve_path(&self, _context_path: &str, path_to_resolve: &Cursor) -> Option<String> {
+        Some(path_to_resolve.node().unparse())
```

Comment:
I don't think this is important because the files being tested are unrelated, but for correctness this should remove the quotes surrounding the string literal received.